fgrpc/grpcrunner.go

```diff
@@ -156,9 +160,16 @@ func RunGRPCTest(o *GRPCRunnerOptions) (*GRPCRunnerResults, error) {
 	return &total, nil
 }
 
-// GRPCDestination parses dest and returns dest:port based on dest type
+// parseGRPCDestination parses dest and returns dest:port based on dest type
```

Comment:
why the name/visibility  change ?

Reply:
since its not being used outside the package anymore and its being embedded into the Dial method

Reply:
go style would probably be to call this grpcDestination() then but it's ok

Reply:
changed to `grpcDestination`